### PR TITLE
Enable test_unary_ufuncs

### PR DESCRIPTION
**Pull request by yuchengliu1**

### CODE REVIEW

File: test/xpu/run_test_with_skip.py [hunk 1]
```diff
@@ -1277,6 +1277,46 @@ def launch_test(test_case, skip_list=None, exe_list=None):
 )
 res += launch_test("nn/test_pooling_xpu.py", skip_list)
 
+skip_list=(
+    # AssertionError: Jiterator is only supported on CUDA and ROCm GPUs, none are available.
+    "_jiterator_",
+
+    # AssertionError: Tensor-likes are not close!
```
Comment (fengyuan14) [inline test/xpu/run_test_with_skip.py:1284]: What's the justification of these skips? CPU fallback fails?

**Comment by yuchengliu1** [comment]
exp seems still failed
There is a example
```
AssertionError: Tensor-likes are not close!

Mismatched elements: 27 / 81 (33.3%)
Greatest absolute difference: nan at index (27,) (up to 1e-05 allowed)
Greatest relative difference: nan at index (27,) (up to 1.3e-06 allowed)

To execute this test, run the following from the base repo dir:
    PYTORCH_TEST_WITH_SLOW=1 python test/test_unary_ufuncs.py -k TestUnaryUfuncsXPU.test_reference_numerics_extremal_exp_xpu_complex64

This message can be suppressed by setting PYTORCH_PRINT_REPRO_ON_FAILURE=0
```

**Comment by fengyuan14** [comment]
> test_reference_numerics_extremal_exp_xpu_complex64

@huaiyuzh Does the case fail by an error about
`std::exp(std::log<std::complex<float>>(extremal_val, extremal_val))`?

File: test/xpu/run_test_with_skip.py [hunk 2]
```diff
@@ -1607,6 +1607,56 @@ def launch_test(test_case, skip_list=None, exe_list=None):
...
+    # For extreme value processing, Numpy and XPU results are inconsistent
+    "test_reference_numerics_extremal__refs_log_xpu_complex64",
+    "test_reference_numerics_extremal_log_xpu_complex64",
+    "test_reference_numerics_extremal__refs_tanh_xpu_complex128",
+    "test_reference_numerics_extremal__refs_tanh_xpu_complex64",
+    "test_reference_numerics_extremal_tanh_xpu_complex128",
+    "test_reference_numerics_extremal_tanh_xpu_complex64",
+
+# AssertionError: Scalars are not equal!
+    "test_exp_xpu_complex128",
```
Comment (fengyuan14) [inline test/xpu/run_test_with_skip.py:1651]: What's the failure? We implemented the operator.

File: test/xpu/run_test_with_skip.py [hunk 3]
```diff
@@ -1607,6 +1607,56 @@ def launch_test(test_case, skip_list=None, exe_list=None):
...
+    "test_exp_xpu_complex64",
+# AssertionError: Tensor-likes are not close!
+    "test_reference_numerics_extremal__refs_exp_xpu_complex128",
```
Comment (fengyuan14) [inline test/xpu/run_test_with_skip.py:1654]: Please classify these failures to the set by `For extreme value processing, Numpy and XPU results are inconsistent`

**Review by fengyuan14 — APPROVED** [review]

**Pull request merged**